terraform/node_output.go

```diff
@@ -9,6 +9,98 @@ import (
 	"github.com/hashicorp/terraform/lang"
 )
 
+// NodePlannableOutput is the placeholder for an output that has not yet had
+// its module path expanded.
+type NodePlannableOutput struct {
+	Addr   addrs.OutputValue
+	Module addrs.Module
+	Config *configs.Output
+}
+
+var (
+	_ GraphNodeSubPath       = (*NodePlannableOutput)(nil)
+	_ RemovableIfNotTargeted = (*NodePlannableOutput)(nil)
+	_ GraphNodeReferenceable = (*NodePlannableOutput)(nil)
+	//_ GraphNodeEvalable          = (*NodePlannableOutput)(nil)
```

Comment:
I don't think `NodePlannableOutput` needs to be Eval-able, this will be cleaned up later if we don't need it.